src/monitoring.tbasicu

```diff
@@ -0,0 +1,167 @@
+Uses "Biff"
+
+#include once "logger_ps.tbasicu"
+
+%DIFFICULTY_MONITOR_RATE = 0.5 's
+%DIFFICULTY_STORE_RATE = 2 ' s (averaging)
+
+Type monitorData
+  temps as Double
+  difficulty as Double
+  meanFrameRate as dword
+  meanDistance as Double
+  meanCapacity as double
+end type
+
+
+Type difficultyMonitor
+
+  contextPtr as dword
+  logger as LoggerPS
+  difficulty as double
+  oldTime as Double ' to calculate time diff between two calls
+  tempsMonitor as double
+  tempsStore as Double
+  rateCount as DWord
+  '[t] can't use undimensioned array conjointly with " Local context as GameContext at me.contextPtr "
```

Comment:
it seems to me that's not in [the list ](https://thinbasic.github.io/book-thinbasic_user_defined_types/appendix-list-of-known-issues.html). In your opinion, is it an issue ? (also see line 86 below)